cfme/fixtures/nuage.py

```diff
@@ -45,9 +44,18 @@ def with_nuage_sandbox(networks_provider):
     box['l2_group'] = box['l2_domain'].create_child(
         nuage.vspk.NUPolicyGroup(name=random_name()))[0]
 
+    return box
+
+
+@pytest.fixture
+def with_nuage_sandbox(networks_provider):
+    nuage = networks_provider.mgmt
+    sandbox = create_nuage_sandbox(networks_provider)
```

Comment:
Please pass `nuage` handle directly to the `create_nuage_sandbox` function instead passing `networks_provider` fixture. We want the auxilary function be as pure as possible.